defaults/main.yml

```diff
@@ -3,7 +3,8 @@
 wimpy_project_name: "{{ wimpy_project | regex_replace('[^a-zA-Z0-9]', '-') }}"
 wimpy_github_project_url: ""
 
-wimpy_docker_image_name: "{{ wimpy_project_name }}" # Name of the docker image that will be created
+wimpy_docker_registry: "{{ wimpy_aws_account_id ~ '.dkr.ecr.' ~ wimpy_aws_region ~ '.amazonaws.com/' if wimpy_aws_account_id is defined else '' }}"
+wimpy_docker_image_name: "{{ wimpy_docker_registry }}{{ wimpy_project_name }}" # Name of the docker image that will be created
```

Comment:
This means that by default if `wimpy_aws_account_id` is set, Wimpy will use Amazon ECR to store docker image. 
But if someone wants to use Dockerhub (for example) even setting `wimpy_aws_account_id`, he should override `wimpy_docker_registry` with an empty value, right?